src/utilities/logging.ts

```diff
@@ -1,7 +1,7 @@
-import { Consola, LogLevel }  from "consola";
+import { Consola, LogLevel } from "consola";
 
 // Default level is Info
-const logger = new Consola({level: 3});
+const logger = new Consola({ level: 3 });
```

Comment:
nit: we should use a const to label this input - something like `const LOG_LEVEL_WARNING = 3;`

Reply:
Looking at this now we can actually just avoid the constant altogether and use the `LogLevel` enum in the Consola package